frontend/src/pages/TermPlanner/ValidateTermPlanner.js

```diff
@@ -25,6 +26,34 @@ const validateTermPlanner = (payload) => {
   };
 };
 
+// takes in startYear (int) and gets current date to return the most recent term that has past (week 2)
+// dictionary of year in degree and term most recent is returned 
+const getMostRecentPastTerm = (startYear) => {
```

Comment:
Sorry to make you change stuff again, but maybe we push this whole function into constants? Since the constants dont make much sense without the constants in that function anyways.

Reply:
Im just terrified of this function - its the type of function that would take me 4h to debug if something is wrong with it.

Reply:
😆 no worries at all, sure I'll move it